### PR TITLE
provider: Support default tags (resources aws_k*)

### PR DESCRIPTION
**Pull request by breathingdust**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Reference: #7926

Output from acceptance testing: Will be handled via CI, review, and daily acceptance testing

### CODE REVIEW

**Review by ewbankkit — APPROVED** [review]
LGTM 🚀.

File: aws/resource_aws_kms_external_key.go [hunk 1]
```diff
@@ -82,6 +82,7 @@ func resourceAwsKmsExternalKey() *schema.Resource {
 				),
 			},
 			"tags": tagsSchema(),
+			"tags_all": tagsSchemaComputed(),
```
Comment (anGie44) [inline aws/resource_aws_kms_external_key.go:85]: @breathingdust just needs the `CustomizeDiff` func here in the schema

File: aws/resource_aws_kms_key.go [hunk 2]
```diff
@@ -97,6 +97,8 @@ func resourceAwsKmsKey() *schema.Resource {
 
 func resourceAwsKmsKeyCreate(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*AWSClient).kmsconn
+	defaultTagsConfig := meta.(*AWSClient).DefaultTagsConfig
```
Comment (anGie44) [inline aws/resource_aws_kms_key.go:100]: also needs `tags_all` and the `CustomizeDiff: SetTagsDiff` in the schema

**Review by anGie44 — APPROVED** [review]
🚀 🚀 🚀

**Pull request merged**